spec/lib/facilities/vha_facility_spec.rb

```diff
@@ -116,14 +116,14 @@
 
       context 'with services' do
         let(:satisfaction_data) do
-          fixture_file_name = ::Rails.root.join("/spec/fixtures/facility_access/satisfaction_data.json")
+          fixture_file_name = ::Rails.root.join('/spec/fixtures/facility_access/satisfaction_data.json')
           File.open(fixture_file_name, 'rb') do |f|
             JSON.parse(f.read)
           end
         end
 
         let(:wait_time_data) do
-          fixture_file_name = ::Rails.root.join("/spec/fixtures/facility_access/wait_time_data.json")
+          fixture_file_name = ::Rails.root.join('/spec/fixtures/facility_access/wait_time_data.json')
```

Comment:
Is it necessary to prepend the scope resolution operator? Could we get rid of the noise and use `Rails.root...` instead of `::Rails.root...`?